File: Tareas/Tarea9/Tarea9.java
/*  
Nombre: Axel Ramon Ortiz Flores Matricula: 1819000 Hora: V6
Tarea 9: Conversion de monedas
*/
import javax.swing.JOptionPane;
public class Tarea9{
	public static void main(String[] args) {
		String entrada;
		String[] options ={"Ninguna", "A Dolares", "A Euros\n", "A Yenes", "A Dolares de Hong Kong"};
		int o;
		float p;
		double d=0.040, e=0.037, y=4.34, dh=0.31, r;

		o = JOptionPane.showOptionDialog(null,"Ingrese la opcion deseada:", "Conversion de Moneda", 
		   JOptionPane.YES_NO_CANCEL_OPTION, JOptionPane.QUESTION_MESSAGE, null, options, options[0]);

		if (o==0) {
			JOptionPane.showMessageDialog(null, "No ha elegido ninguna opcion, GRACIAS!!");
		}else{

		entrada= JOptionPane.showInputDialog("Ingrese la cantidad de pesos a convertir: ");
		p= Float.parseFloat(entrada);


		switch(o){

			case 1: r= p*d;
			        JOptionPane.showMessageDialog(null, "La cantidad en dolares es: "+String.format("%.3f", r));
			        break;

			case 2: r= p*e;
			        JOptionPane.showMessageDialog(null, "La cantidad en euros es: "+String.format("%.3f", r));
			        break;

			case 3: r=p*y;
			        JOptionPane.showMessageDialog(null, "La cantidad en yenes es: "+String.format("%.3f", r));
			        break;

			case 4: r= p*dh;
			        JOptionPane.showMessageDialog(null, "La cantidad en dolares de Hong Kong es: "+String.format("%.3f", r));
			        break;

		    }
		}
	}
}

File: Tareas/Tarea9/Tarea9.rb
=begin 
Nombre: Axel Ramon Ortiz Flores  Matr.1819000  Hora:V6
Tarea 9: Conversión de monedas
=end


print("\t .: Menu :.\n\nConversion de Pesos a:\n\t1.Dolar\n\t2.Euro\n\t3.Yen(Japones)\n\t4.Dolar de Hong Kong\n\n")
print("Del Menu anterior, ingrese la conversion que desea realizar: ")
n= gets().to_i

	case n
		when 1
			print("\nIngrese la cantidad de Dolares a convertir: ")
        	x= gets().to_f
        	P=x*23.12
        	puts("\n#{x} Dolares es igual a #{P} Pesos")
	
		when 2
			print("\nIngrese la cantidad de Euros a convertir: ")
        	x= gets().to_f
        	P=x*25.41
        	puts("\n#{x} Euros es igual a #{P} Pesos")
		when 3
			print("\nIngrese la cantidad de Yenes a convertir: ")
        	x= gets().to_f
        	P=x*0.22
          	puts("\n#{x} Yenes es igual a #{P} Pesos")
	
		when 4
			print("\nIngrese la cantidad de Dolares de Hong Kong a convertir: ")
        	x= gets().to_f
        	P=x*2.99
        	puts("\n#{x} HDK es igual a #{P} Pesos")

		else
			puts("\n\n Debe ingresar un numero de acuerdo al menu dado anteriormente!!")
	end